public/templates/comments/comments.tpl

```diff
@@ -80,6 +80,13 @@
                                             </a>
                                         </li>
                                         <!-- ENDIF !posts.selfPost -->
+                                        <!-- IF isAdmin -->
```

Comment:
i see that this is onyl for admin? 

tciket description:
As a set owner I am able to delete every comment

As a comment owner I am only able to delete my own comment I am not able to delete other users' comments unless I am set's owner,

As an admin I am able to delete every comment there is,

Reply:
Nope it's not only for admin. There's separate part for the post owner just above added lines.